bids-validator/validators/json/schemas/pet.json

```
@@ -0,0 +1,215 @@
+{
+  "type": "object",
+  "properties": {
+    "Manufacturer": { "type": "string", "minLength": 1 },
+    "ManufacturersModelName": { "type": "string", "minLength": 1 },
+    "Unit": {
+      "type": "string",
+      "minLength": 1
+    },
+    "TracerName": { "type": "string", "minLength": 1 },
+    "TracerRadionuclide": { "type": "string", "minLength": 1 },
+    "InstitutionName": { "type": "string", "minLength": 1 },
+    "InstitutionAddress": { "type": "string", "minLength": 1 },  
+    "InstitutionDepartmentName": { "type": "string", "minLength": 1 }, 
```

Fix typo from https://github.com/bids-standard/bids-specification/pull/771#issue-612244722

```suggestion
    "InstitutionalDepartmentName": { "type": "string", "minLength": 1 }, 
```